### ISSUE TITLE
`get_authorization_url` and other built urls does not work in release 5.0.0

### ISSUE THREAD

**Issue by AbundantSalmon** [workos/workos-python#341]
On release 5.0.0, `get_authorization_url(...)` does not work, in fact anything that uses the internal `RequestHelper.build_url_with_query_params` does not work.

```python
    authorisation_url = workos_client.user_management.get_authorization_url(
        provider="authkit", redirect_uri=redirect_uri, state=str(uuid.uuid4())
    )

   # ` authorisation_url` is https://api.workos.com/?client_id=XXX&redirect_uri=XXX&response_type=code&provider=authkit&state=XXX
   # instead of https://api.workos.com/user_management/authorize?client_id=XXX&redirect_uri=XXX&response_type=code&provider=authkit&state=XXX
```

This is due to the `build_url_with_query_params` function expecting `base_url` to be in the format `https://api.workos.com/{}` but it provided as `https://api.workos.com/`.

If I try to modify your codebase like below it breaks a bunch of tests, so will leave it up to you guys to resolve your code. Hope my detailed response is helpful!

```python
class RequestHelper:

    @classmethod
    def build_parameterized_url(cls, url: str, **params: QueryParameterValue) -> str:
        escaped_params = {k: urllib.parse.quote(str(v)) for k, v in params.items()}
        return url.format(**escaped_params)

    @classmethod
    def build_url_with_query_params(
        cls, base_url: str, path: str, **params: QueryParameterValue
    ) -> str:
        # return base_url.format(path) + "?" + urllib.parse.urlencode(params)
        return base_url + path + "?" + urllib.parse.urlencode(params)
```

**Comment by BoatPartyJesus** [comment]
We are also experiencing this issue when moving to 5.0

**Comment by PaulAsjes** [comment]
Hey both, thanks for the report. We're working on a fix and hope to have it released shortly.

**Comment by PaulAsjes** [comment]
This has been fixed in v5.0.1!

**Comment by AbundantSalmon** [comment]
Thanks!
